File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/AudienceResponse.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class AudienceResponse 
{
    public long Id { get; set; }
    public string Name { get; set; }
    public double Latitude { get; set; }
    public double Longitude { get; set; }
    public double Radius { get; set; }
    public bool SendSuggestion { get; set; }
    public string Description { get; set; }
    public bool Active { get; set; }
    public string SearchTags { get; set; }
    public long OwnerId { get; set; }
    public string Gender { get; set; }
    public List<AgeGroupResponse> AgeGroups { get; set; }
    public List<CategoryResponse> Categories { get; set; }
    public List<ApplicationShortResponse> Applications { get; set; }
    public List<AudienceDeviceResponse> Devices { get; set; }
    public string GameExperienceLevel { get; set; }
    public List<GeocodeLatLngResponse> Locations { get; set; }
    public string AssociateType { get; set; }
    public long AssociateId { get; set; }
    public string AssociateDescription { get; set; }
}

File: server-scala-play-server/app/model/AudienceResponse.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for AudienceResponse.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class AudienceResponse(
  id: Option[Long],
  name: Option[String],
  latitude: Option[Double],
  longitude: Option[Double],
  radius: Option[Double],
  sendSuggestion: Option[Boolean],
  description: Option[String],
  active: Option[Boolean],
  searchTags: Option[String],
  ownerId: Option[Long],
  gender: Option[String],
  ageGroups: Option[List[AgeGroupResponse]],
  categories: Option[List[CategoryResponse]],
  applications: Option[List[ApplicationShortResponse]],
  devices: Option[List[AudienceDeviceResponse]],
  gameExperienceLevel: Option[String],
  locations: Option[List[GeocodeLatLngResponse]],
  associateType: Option[String],
  associateId: Option[Long],
  associateDescription: Option[String]
)

object AudienceResponse {
  implicit lazy val audienceResponseJsonFormat: Format[AudienceResponse] = Json.format[AudienceResponse]
}

